### PR TITLE
Feature/11999 add link to accessability document

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Add Link to Accessability Document

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11999

## Screenshots
Simulator Screen Shot - iPhone 11 Pro Max - 2022-02-28 at 03 41 26

### CODE REVIEW

**Comment by marcussc** [comment]
Simulator Screen Shot - iPhone 13 mini - 2022-02-28 at 09 48 03
Seems like the image is not scaling but the chevrons of the other cells are scaling.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.links.strings [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 "safari_corona_social_media" = "https://www.coronawarn.app/de/community/#socialmedia";
 
+"safari_corona_accessibility" = "https://www.coronawarn.app/de/accessability";
```
Comment (KathrinLuecke) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.links.strings:13]:
```suggestion
"safari_corona_accessibility" = "https://www.coronawarn.app/de/accessibility";
```

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.links.strings [hunk 2]
```diff
@@ -10,6 +10,8 @@
 
 "safari_corona_social_media" = "https://www.coronawarn.app/en/community/#socialmedia";
 
+"safari_corona_accessibility" = "https://www.coronawarn.app/en/accessability";
```
Comment (KathrinLuecke) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.links.strings:13]:
```suggestion
"safari_corona_accessibility" = "https://www.coronawarn.app/en/accessibility";
```
Also to be corrected for other languages where EN link is used

**Review by KathrinLuecke — DISMISSED** [review]

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.links.strings (hunk 2 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.links.strings]: @KathrinLuecke I updated it, The typo was in the jira ticket.

**Review by marcussc — APPROVED** [review]

**Review by KathrinLuecke — COMMENTED** [review]
UA approved

**Pull request merged**